primitives/runtime/src/testing.rs

```diff
@@ -33,6 +33,12 @@ use crate::transaction_validity::{TransactionValidity, TransactionValidityError,
 #[derive(Default, PartialEq, Eq, Clone, Encode, Decode, Debug, Hash, Serialize, Deserialize, PartialOrd, Ord)]
 pub struct UintAuthorityId(pub u64);
 
+impl fmt::Display for UintAuthorityId {
```

Comment:
It is totally fine since it is testing code, but I just wonder why? Debug is already derived.

Reply:
Leftover from previous iterations... i removed this one.